### ISSUE TITLE
Date is truncated in Date picker.

### ISSUE THREAD

**Issue by AgneLukoseviciute** [microsoft/react-native-gallery#234]
Found during accessibility grading.

Tracking internal bug: https://microsoft.visualstudio.com/DefaultCollection/OS/_workitems/edit/40139451
"Users with low vision will not able to consume the content of the button if it is not visible in Text scaling mode."

**Comment by AgneLukoseviciute** [comment]
![image](https://user-images.githubusercontent.com/41223743/175647081-c00e5cae-acf0-41e1-b41a-d4ee21f2aa4a.png)
